stable/oauth2-proxy/values.yaml

```diff
@@ -60,6 +60,8 @@ authenticatedEmailsFile:
 
 service:
   type: ClusterIP
+  # when service type is ClusterIP a valid param is `clusterIP` -> https://kubernetes.io/docs/concepts/services-networking/service/#choosing-your-own-ip-address
+  # when service type is LoadBalancer a valid param is `loadBalancerSourceRanges`, which defaults to "0.0.0.0/0" if not specified
   port: 80
   annotations: {}
   # foo.io/bar: "true"
```

Comment:
I think it would be clearer to have a commented example like
```
# when service.type is ClusterIP ...
# clusterIP: 192.0.2.20
# when service.type is LoadBalancer ...
# loadBalancerIP: 198.51.100.40
# loadBalancerSourceRanges: 203.0.113.0/24
```

Reply:
yup, looks cleaner!, thanks and done